src/main/java/ToDo.java

```diff
@@ -0,0 +1,26 @@
+/** ToDo Class Represents a ToDo task that contains a description */
+public class ToDo extends Task {
+
+  /**
+   * Constructor that takes in the task description
+   *
+   * @param description task description
+   */
+  public ToDo(String description) {
+    super(description);
+  }
+
+  /**
```

Comment:
Perhaps you can consider adding an `@Override` to show that this method overrides getType() in `Task`?